core/src/blocks/map.rs

```diff
@@ -105,10 +105,10 @@ impl Block for Map {
                             ))?;
                         }
                         match arr.len() {
```

Comment:
You don't need the match anymore here right?

Reply:
Indeed, lemme change it